source/simulator/parameters.cc

```diff
@@ -757,20 +757,26 @@ namespace aspect
                          Patterns::Integer (0),
                          "The number of adaptive refinement steps performed after "
                          "initial global refinement but while still within the first "
-                         "time step.");
+                         "time step. These refinement steps (n) are added to the value "
+                         "for initial global refinement (m) so that the total "
+                         "refinement level is n+m.   ");
```

Comment:
That's not necessarily true because a refinement step may not actually create a new level. Maybe say "so that the final mesh has cells that are at most on refinement level $n+m$."

No need for the spaces at the end. Extra points if you mark up the `n` and `m` in previous lines as `$math$`.